packages/core/src/components/OverflowTooltip/OverflowTooltip.tsx

```diff
@@ -97,22 +96,19 @@ export const HvOverflowTooltip = (props: HvOverflowTooltipProps) => {
     ]
   );
 
-  return open || isOverflowing ? (
+  return (
     <HvTooltip
       id={id}
       disableHoverListener={!isOverflowing}
       open={open}
       placement={placement}
-      title={
-        <HvTypography className={classes.tooltipData} variant="body">
-          {data}
-        </HvTypography>
-      }
+      title={data}
```

Comment:
I don't really like the `HvTypography` here too since data is a `React.Node`. However, by doing this we lose the classes `tooltipData` so if someone is using it to style the title it won't work anymore.

Reply:
You're right, I overlooked this 😞

We're using `HvTypography` to override MUI text styles... but they're already being overriden, except for the `fontFamily`